projects/storefrontlib/src/cms-components/cart/add-to-cart/add-to-cart.module.ts

```diff
@@ -42,6 +42,9 @@ import { AddedToCartDialogComponent } from './added-to-cart-dialog/added-to-cart
       cmsComponents: {
         ProductAddToCartComponent: {
           component: AddToCartComponent,
+          data: {
+            inventoryDisplay: 'true',
```

Comment:
I would like it to be in a maybe the viewConfig to enable and disable this feature as I believe it falls under the 'view'.

anyhow, couldn't this have been `true` instead of a string 'true'

Reply:
Uses regular configuration now.